client/scss/components/forms/_publishing.scss

```diff
@@ -0,0 +1,11 @@
+// Styles for the fields in the scheduled publishing dialog
+.w-panel.publishing {
+  .w-panel__wrapper {
+    margin-top: theme('spacing.2');
+    margin-bottom: theme('spacing.2');
+  }
+
+  .w-field--date_time_field input {
+    width: 100%;
```

Comment:
Without this, the field row overflows the parent's width. Tried various things but this is the only way I got it to work...